ibis/backends/tests/test_timecontext.py

```diff
@@ -95,7 +95,7 @@ def test_context_adjustment_window_udf(
     """ This test case aims to test context adjustment of
         udfs in window method.
     """
-    with option_context('time_col', 'timestamp_col'):
+    with option_context('timecontext.time_col', 'timestamp_col'):
```

Comment:
should be under a the namespace context_adjustment

Reply:
How about `timecontext_adjustment`? We have been calling this `timecontext` instead of `context` all over the place

Reply:
Use context_adjustment as suggested